docs/sources/send-data/promtail/_index.md

```diff
@@ -3,7 +3,7 @@ title: Promtail agent
 menuTitle:  Promtail
 description: How to use the Promtail agent to ship logs to Loki
 aliases: 
-- ../clients/promtail/
+- ../../clients/promtail/
```

Comment:
I think it's worthwhile to include a comment explaining the absolute URL path to help with validation and making future changes easier to reason about.

I think this should only be one set of `..`. I generated the following relative alias using the user script for relative aliases: https://github.com/grafana/website/blob/master/scripts/user-scripts/relative-aliases.user.js

```suggestion
- ../clients/promtail/ # /docs/loki/latest/clients/promtail/
```